src/Components/Blueprints/BuildImagesButton.tsx

```diff
@@ -120,11 +120,7 @@ export const BuildImagesButton = () => {
                       />
                     </FlexItem>
                   )}
-                  <FlexItem>
-                    {deselectedTargets.length !== 0
-                      ? 'Build selected'
-                      : 'Build image'}
-                  </FlexItem>
+                  <FlexItem>{'Build images'}</FlexItem>
```

Comment:
Just one tiny nit pick :nerd_face: 

```suggestion
                  <FlexItem>Build images</FlexItem>
```

Reply:
Looks like some tests will need a small update